### ISSUE TITLE
Feature Request: GroovyMiSTer

### ISSUE THREAD

**Issue by psakhis** [theypsilon/Update_All_MiSTer#129]
Hi!
I wanted to ask you if it would be possible to add GroovyMiSTer as an element of option "Other Cores" submenu.

Maybe the most complicated thing would be parse MiSTer.ini to add 2 lines for start custom GroovyMiSTer binary.

Even more complicated would be to add XDP version. It needs custom kernel an another 2 files on /usr/lib

Where would I need to place the files in git to automate the process? 

Thanks for this great tool.

**Comment by Numbski** [comment]
I don't think any work has gone into GroovyMAME on DE10.  Mostly commenting in case I am wrong, but by all accounts the ARM CPU on the DE-10 is a bit gutless compared to say a Raspberry Pi.

Tony Shadwick

**Comment by psakhis** [comment]
> I don't think any work has gone into GroovyMAME on DE10. Mostly commenting in case I am wrong, but by all accounts the ARM CPU on the DE-10 is a bit gutless compared to say a Raspberry Pi. Tony Shadwick

GroovyMiSTer isn't GroovyMAME. GroovyMiSTer is a fpga core and acts like analog gpu.

Software emulation like GroovyMAME or any other software like MiSTerCast runs on PC side and fpga receives frames from DE10 ethernet.

**Comment by Numbski** [comment]
You're right, I had forgotten about that.  In that case, this sounds like a great thing to ask about.

Tony Shadwick
